packing_manager_specs/RPM/rpmbuild/SPECS/gz-beat-shipper.spec

```diff
@@ -0,0 +1,33 @@
+%define name    gz-beat-shipper
+%define version 0.1
+%define _rpmdir %{getenv:GITHUB_WORKSPACE}/
+
+Name:           %{name}
+Version:        0.0.1
```

Comment:
This and the one defined above do not match.

Reply:
I didn't realize, thanks. I've used the variable.